firedantic/_async/model.py

```diff
@@ -38,7 +38,7 @@
 }
 
 
-def get_collection_name(cls, name: Optional[str]) -> str:
+def get_collection_name(cls, name: Optional[str] = None) -> str:
```

Comment:
You missed one of the two instances:
![image](https://github.com/ioxiocom/firedantic/assets/6637576/05ffa968-8f4d-42d2-b1f1-883dd0ca6a1d)

Reply:
Sorry, I missed this was still pending

Reply:
Oh, and my bad, it was resolved, but I got confused when I accidentally opened the old version of the file instead of the new.